cpp/src/slice2matlab/Main.cpp

```diff
@@ -2658,71 +2658,39 @@ CodeVisitor::visitExceptionStart(const ExceptionPtr& p)
     //
     // Constructor
     //
-    out << nl << "function " << self << " = " << name << spar << "ice_exid"
-        << "ice_exmsg" << allNames << epar;
+    out << nl << "function " << self << " = " << name << spar << "errID" << "msg" << epar;
```

Comment:
We still need to generate a constructor in MATLAB to fill-out errID + msg because the "activator" constructs the exception class with no argument.